### DISCUSSION TITLE
SafePal wallet is not working properly

### DISCUSSION THREAD

**Discussion by VasilVR** [WalletConnect/walletconnect-monorepo#5215] [Developer Support]
Hello WalletConnect team!

There is a problem when I connect to my dApp with SafePal using WalletConnect. I'm using ethers. The problem is the transactions are executed on the blockchain but the response appears on my dApp after a few minutes. Meanwhile, it's just loading. I don't have this problem if I'm using MetaMask for example. I set the correct project ID from WalletConnect, the domain is also verified. I tried with custom paid RPC but it's still the same. I got this error when I connect to the dapp as well.
![screen](https://github.com/user-attachments/assets/25ef3c83-55d6-426f-91ca-67830d7f4dbd)

**Comment by safepal-support-16313090221** [comment]
I fallen with same problem .I contacted 1 631 309 0221. They fixed my issues and charged me USD 30. Probably Name of the guy was Alby .Trust Person u must try . I think best for "dapp support number 1 631 309 0221 .
I tried to contact others too after long wait i connected with SafePal wallet Contact  number 1 631 309 0221.
